lib/fastlane/plugin/appcenter/actions/appcenter_upload_action.rb

```diff
@@ -2,6 +2,18 @@ module Fastlane
   module Actions
     module Constants
       MAX_RELEASE_NOTES_LENGTH = 5000
+      SUPPORTED_EXTENSIONS = {
+          android: %w(.aab .apk),
+          ios: %w(.ipa),
+          mac: %w(.app .app.zip .dmg .pkg),
```

Comment:
The portal does not have .app on the support list. Whether it is required to be ".app.zip" or any .zip is fine also depends on the implementation of this code; it's handled differently in different places.

Reply:
The portal supports .app and automatically zips for you.

Reply:
Nevermind my comment. macOS does this for you (at least Chrome on macOS).
But anyway the plugin does this automatic zipping for you :)